### PR TITLE
Add nuspec template file

### LINKED ISSUE #44: Add nuspec file
To the folder `.ci/BHoMBot/Nuget` - in the form of `{repository_name}.nuspec` to help the bot automatically produce nugets of this repository.

### PR DESCRIPTION
**Pull request by FraserGreenroyd**
Fixes #44

### CODE REVIEW

**Review by kprusicka — DISMISSED** [review]
Looks amazing great work

File: .ci/BHoMBot/Nuget/BHoM.Interop.SQL.nuspec [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <icon>icon.png</icon>
+    <readme>readme.md</readme>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Interoperability with SQL databases provided by the SQL_Toolkit of the Building and Habitats object Model (BHoM) - more information available at https://github.com/BHoM/SQL_Toolkit</description>
+    <releaseNotes></releaseNotes>
+    <copyright></copyright>
+    <tags>BHoM interop sql aec</tags>
+    <title></title>
+    <dependencies>
+      <group targetFramework=".NetStandard2.0">
```
Comment (FraserGreenroyd) [inline .ci/BHoMBot/Nuget/BHoM.Interop.SQL.nuspec:18]:
```suggestion
      <group targetFramework="netstandard2.0">
```

**Review by kprusicka — APPROVED** [review]
Looks even better

**Review by Dhorix — APPROVED** [review]
Reviewed, looks great

**Comment by FraserGreenroyd** [comment]
@BHoMBot this is a DevOps instruction. I am requesting neutral checks on: code-compliance, copyright-compliance, core, documentation-compliance, installer, project-compliance, ready-to-merge, serialisation, versioning, dataset-compliance, unit-tests

**Comment by FraserGreenroyd** [comment]
@BHoMBot this is a DevOps instruction. I am requesting neutral checks on: null-handling

**Pull request merged**